src/frontend/hooks/useLocation.ts

```diff
@@ -118,3 +142,24 @@ export function getCoords(location: LocationObject): [number, number] {
   const {longitude, latitude} = location.coords;
   return [longitude, latitude];
 }
+
+export function getAccuracy(
+  previousAccuracy: number | null,
+  newAccuracy: number | null,
+  distance: number,
+): Accuracy {
+  if (previousAccuracy === null) {
+    return newAccuracy === null ? 'unchanged' : 'better';
+  }
+
+  if (distance > previousAccuracy) {
+    return 'newBounds';
+  }
```

Comment:
It could be the case that the user has travelled a large distance, but the accuracy has not gotten better. If the user is asking for the most "accurate" location, we would return the value with the lowest accuracy radius (which in this case is not actually the most accurate). This check will allow the user to know if they have travelled outside of the original accuracy bounds.